dashboard/src/app/factories/factories-config.ts

```diff
@@ -52,12 +52,7 @@ export class FactoryConfig {
           title: 'Load Factory',
           templateUrl: 'app/factories/load-factory/load-factory.html',
           controller: 'LoadFactoryController',
-          controllerAs: 'loadFactoryController',
-          resolve: {
-            initData: ['factoryConfigService', (svc: FactoryConfigService) => {
-              return svc.allowFactoriesRoutes();
-            }]
-          }
+          controllerAs: 'loadFactoryController'
```

Comment:
Wow. Seems we need to distinct persistedFactories and creating workspace via factory scenario with URL...
Plan minimum - make it clearer in [README.md](https://github.com/eclipse/che/tree/master/dashboard#configurability) what each feature responsible for.
Plan maximum - think about better naming that would allow us to distinct persisted factories vs load-factory with URL

@olexii4 @ibuziuk WDYT?

Reply:
Until I saw what the `/f?=url` replaced to in the address bar, I assumed `f` stood for "file" as in devfile :)

Reply:
@sleshchenko I have done with [README.md](https://github.com/eclipse/che/tree/master/dashboard#configurability).  I think the best variant will be a separate flow for creating a new workspace from the devfile with their own routing. It can be ```/d?=url``` or  ```/file?=url```.